src/sentry/issues/endpoints/organization_group_index.py

```diff
@@ -204,7 +206,7 @@ def use_group_snuba_dataset():
                     "search.use_group_snuba_dataset", query_kwargs["use_group_snuba_dataset"]
                 )
 
-                result = search.query(**query_kwargs)
+                result = search.query(**query_kwargs)  # type: ignore[attr-defined]
```

Comment:
@asottile-sentry what's the right way to type this?

Reply:
https://www.notion.so/sentry/sentry-typing-guidelines-best-practices-471d5aacf262476188982ce70f909e00?pvs=4#7a338a289f3541c38ad79ea6666140ee

Reply:
`error: Module has no attribute "backend"` 

we don't actually expose backend [here](https://github.com/getsentry/sentry/blob/1e6937c8b51e507e105f95df7ceb290a6ca99afc/src/sentry/search/__init__.py#L7) -- should that be 

```
backend = LazyServiceWrapper(
    SearchBackend, settings.SENTRY_SEARCH, settings.SENTRY_SEARCH_OPTIONS
)
backend.expose(locals())
```

Reply:
yeah if we don't have one there it should be